### PR TITLE
#184 Update PR info in ReadMe

### LINKED ISSUE #184: Add new points for PR in Read Me

### PR DESCRIPTION
**Pull request by kmaslowiec**
#184

### CODE REVIEW

**Review by BorysChajdas — APPROVED** [review]

**Review by marcinWisnia — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -127,8 +127,11 @@ Example:
 
 ### Pull Requests
 
-- Pull Request title should be the same as commit message
+- Pull Request title should be the same as the commit message
 - Only one commit should be present in one Pull Request
+- Progress not perfection. In the review, we suggest what can be improved. If the issue does not affect the functionality it is up to the owner of the PR if he wants to implement it.
+- All comments need to be answered before the approval.
+- If you approve the PR and you have not found an issue please try to leave a comment.
```
Comment (MarekMacko) [inline README.md]: So if everything is perfect for me, then besides the approve I need to leave a comment? Could you give an example? 
e.g.  "Everything is ok for me" ?

Comment (michal-kucznerowicz) [reply README.md]: IMO giving approve means "your changes look good" by itself, thus the additional comment will be redundant.

Comment (MarekMacko) [reply README.md]: @michal-kucznerowicz  That's my point ☝️

File: README.md [hunk 2]
```diff
@@ -127,8 +127,11 @@ Example:
 
 ### Pull Requests
 
-- Pull Request title should be the same as commit message
+- Pull Request title should be the same as the commit message
```
Comment (MarekMacko) [inline README.md:130]: Please check this point in your PR

**Review by MarekMacko — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kmaslowiec) [reply README.md]: Fine with me.

File: README.md (hunk 2 above)
Comment (kmaslowiec) [reply README.md:130]: Fixed. Thank you.

Comment (kmaslowiec) [reply README.md:130]: @MarekMacko @michal-kucznerowicz The requirement has been removed.

Comment (MarekMacko) [reply README.md:130]: I don't get it, requirement removed? Requirement is still in place

Comment (kmaslowiec) [reply README.md:130]: Not this comment.

File: README.md (hunk 1 above)
Comment (kmaslowiec) [reply README.md]: @MarekMacko @michal-kucznerowicz The requirement has been removed.

**Review by michal-kucznerowicz — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (MarekMacko) [reply README.md:130]: Never mind, now everything is according to requirement 😄

**Review by MarekMacko — APPROVED** [review]

**Pull request merged**